docs/advanced/document-store/elasticsearch.md

```diff
@@ -92,41 +92,77 @@ da2.summary()
 ```
 
 ```text
-              Documents Summary               
-                                              
-  Length                 2000                 
-  Homogenous Documents   True                 
-  Common Attributes      ('id', 'embedding')  
-                                              
-                      Attributes Summary                       
-                                                               
-  Attribute   Data type      #Unique values   Has empty value  
- ───────────────────────────────────────────────────────────── 
-  embedding   ('ndarray',)   1000             False            
-  id          ('str',)       1000             False            
-                                                               
-              Storage Summary               
-                                            
-  Backend            ElasticSearch          
-  Host               http://localhost:9200  
-  Distance           cosine                 
-  Vector dimension   128                    
-  ES config          {}                     
-                                            
+              Documents Summary
+
+  Length                 2000
+  Homogenous Documents   True
+  Common Attributes      ('id', 'embedding')
+
+                      Attributes Summary
+
+  Attribute   Data type      #Unique values   Has empty value
+ ─────────────────────────────────────────────────────────────
+  embedding   ('ndarray',)   1000             False
+  id          ('str',)       1000             False
+
+              Storage Summary
+
+  Backend            ElasticSearch
+  Host               http://localhost:9200
+  Distance           cosine
+  Vector dimension   128
+  ES config          {}
+
 [0.14890289 0.3168339  0.03050802 0.06785086 0.94719299 0.32490566
  ...]
 ```
 
 Other functions behave the same as in-memory DocumentArray.
 
+### Bulk request configuration
+
+You can configure how bulk requests is being sent to Elasticsearch when adding documents by adding additional `kwargs` when adding documents. Bulk operation is used for several index operations; like (`index` or `delete`). See [the official Documentation](https://elasticsearch-py.readthedocs.io/en/v8.3.3/helpers.html) for more details. Notes that only the following parameters are accepted, the others will be ignored
```

Comment:
This is not configuration. This is customization perhaps

Reply:
Remove reference to what is accepted and what not. User's responsability

Reply:
noted